db/migrate/20230920044543_create_post.rb

```diff
@@ -0,0 +1,14 @@
+class CreatePost < ActiveRecord::Migration[7.0]
+  def change
+    create_table :post do |t|
+      t.references :author_id, null: false, foreign_key: { to_table: :user }
```

Comment:
- [ ] Hi @kamranasif506, I think here we should reference the `author` column of the `users` table in order for Rails to create the author_id as a reference table in the `posts` table. **Hints:** ` t.references :author, foreign_key: { to_table: 'users' }`